app/models/cd_api/hearing_events.rb

```diff
@@ -2,5 +2,9 @@
 
 module CdApi
   class HearingEvents < BaseModel
+    def self.find(id, params)
```

Comment:
This method seems to be tied to hearing_events

Could we rename this to `find_hearing_events` or make the method with > endpoint from: "/v2/...../hearing_events" dynamically change

Reply:
Not sure there would be sufficient reasoning for this. In this instance, we are overwriting the existing find function that HearingEvents inherits from the BaseModel. Changing the method name would lead to inconsistency in our controllers - where normally we just call find, in this instance we would be calling HearingEvents.find_hearing_events